change/@adaptive-web-adaptive-ui-ce7bff83-c7cd-440d-84be-62a08db55e7b.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Technically this is breaking because I've changed the CLI command to `aui`. Did the initial version get published?